### PR TITLE
words: add some cloven-hoofed ruminants

### PR DESCRIPTION
**Pull request by lrewega**
The Bovini tribe is sorely underrepresented.

### CODE REVIEW

**Comment by DentonGentry** [comment]
The DCO error is because we require commits to have a Signed-Off-By line, indicating that the commit is suitable for open source release.

To add one:
```
git commit --amend --signoff
git push -f
```

**Comment by lrewega** [comment]
> The DCO error is because we require commits to have a Signed-Off-By line

The error is really because I was being cheeky and using the GitHub interface rather than doing this the right way 🙃 

Signed off now.

File: words/tails.txt [hunk 1]
```diff
@@ -272,3 +272,12 @@ qilin
...
 mercat
+bison
+buffalo
+gaur
+saola
+anoa
+banteng
+beefalo
+yak
+cow
```
Comment (catzkorn) [inline words/tails.txt:283]: These last two are duplicates, we have a test that checks for duplicates in the lists. Can you run the test and remove any duplicates?

Comment (lrewega) [reply words/tails.txt:283]: Wow, apologies -- GitHub search failed me here.

**Comment by catzkorn** [comment]
(psst, they are still there!)

**Review by catzkorn — APPROVED** [review]
🐄

**Pull request merged**